### ISSUE TITLE
Snowflake / drop column: unparsable section for drop column statement

### ISSUE THREAD

**Issue by myschkyna** [sqlfluff/sqlfluff#1617]
## Expected Behaviour
I expected to be able to lint a SQL file containing a 'alter table ... drop column ...' statement.

## Observed Behaviour
When linting this SQL statement an unparsable section error occurred, see code report below:
```bash
== [test_schema/test_table/drop_test_column.sql] FAIL
L:   2 | P:  12 |  PRS | Line 2, Position 12: Found unparsable section: '
                       | test_column'
```

## Steps to Reproduce
1. Install the SQLFluff package version 0.7.0a4
2. Create a file with content below
3. Run sqlfluff lint --dialect snowflake <path-to-file>

Content drop_test_column.sql:
```sql
ALTER TABLE test_schema.test_table
DROP column test_column;

```

## Dialect
Snowflake

## Version
sqlfluff, version 0.7.0a4

## Configuration
```
[sqlfluff]
dialect = snowflake
exclude_rules = L009,L011,L029,L031,L034

[sqlfluff:rules]
tab_space_size = 2

[sqlfluff:rules:L010]  # Keywords
capitalisation_policy = upper

[sqlfluff:rules:L014]  # Tables, columns, views
extended_capitalisation_policy = lower

[sqlfluff:rules:L030]  # Function names
capitalisation_policy = upper

[sqlfluff:rules:L040]  # Null & Boolean Literals
capitalisation_policy = upper

[sqlfluff:rules:L016] # Max line length
max_line_length = 120
```
